### DISCUSSION TITLE
Feature Request: Pause On Event

### DISCUSSION THREAD

**Discussion by hediet** [RecordReplay/devtools#4083] [Ideas]
Replay has a nice view of events.

![image](https://user-images.githubusercontent.com/2931520/138137745-cc473ad9-e182-4946-a3ba-6873fb7874df.png)

It would be very useful if I could pause the execution of the handler of this (e.g. Key Press) event, so that I can step into the code that (probably) caused the bug I want to debug from there.

Currently the only way to get started with replay seems to be clicking on a console.log output or setting a breakpoint in the source.
If there are thousands of source files and no helpful console.log outputs, it is quite challenging to find the right entry point.

**Comment by jasonLaster** [comment]
You can see the event listeners [here](https://replayio.notion.site/Viewing-Event-Listeners-24076b8aa1ff41d5982573a26148f3f5). It'd be nice to do a better job of connecting the dots though.
